src/main/java/seedu/taskman/storage/XmlSerializableTaskMan.java

```diff
@@ -66,20 +71,37 @@ public UniqueActivityList getUniqueActivityList() {
                 //TODO: better error handling
             }
         }
+        for (XmlAdaptedEvent q : events) {
+            try {
+                lists.add(new Activity(q.toModelType()));
+            } catch (IllegalValueException e) {
+                //TODO: better error handling
+            }
+        }
         return lists;
     }
 
     @Override
     public List<Activity> getActivityList() {
-        return tasks.stream().map(p -> {
-            try {
-                return new Activity(p.toModelType());
-            } catch (IllegalValueException e) {
-                e.printStackTrace();
-                //TODO: better error handling
-                return null;
-            }
-        }).collect(Collectors.toCollection(ArrayList::new));
+        return Stream.concat(tasks.stream().map(p -> {
```

Comment:
Again p. 
Also, you might want to start a new line at  "concat(", so the blocks of code for the streams end up more left. (Not really sure if this will happen but it might make the code more readable)